src/core/atomicdex/utilities/cpprestsdk.utilities.cpp

```diff
@@ -54,3 +54,23 @@ handle_exception_pplx_task(pplx::task<void> previous_task)
 #endif
     }
 }
+
+pplx::task<fs::path> download_file(const t_http_client_ptr& client, const std::string& url, const fs::path& output_location)
+{
+    SPDLOG_DEBUG("Start downloading {} to {}", url, output_location.c_str());
+    return client->request(web::http::methods::GET, url)
```

Comment:
`url` is a string, this is not supported by `cpprestsdk` on windows, you need to use the macro `FROM_STD_STR` i believe.